homeassistant/components/plum_lightpad/config_flow.py

```diff
@@ -0,0 +1,60 @@
+"""Config flow for Plum Lightpad."""
+import logging
+from typing import Any, Dict, Optional
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers import ConfigType
+
+from .const import DOMAIN
+from .utils import load_plum
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PlumLightpadConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Plum Lightpad integration."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: Optional[ConfigType] = None
+    ) -> Dict[str, Any]:
+        """Handle a flow initialized by the user or redirected to by import."""
+        if not user_input:
+            return self.async_show_form(
+                step_id="user",
+                data_schema=vol.Schema(
+                    {
+                        vol.Required(CONF_USERNAME): str,
+                        vol.Required(CONF_PASSWORD): str,
+                    }
+                ),
+            )
+
+        username = user_input[CONF_USERNAME]
+        password = user_input[CONF_PASSWORD]
+
+        # load Plum just so we know username/password work
+        await load_plum(username, password, self.hass)
```

Comment:
How will you know if username/password are correct?

Reply:
Can we tell that to the user via errors?

Reply:
Thanks for suggestion - added support via errors